src/tokens/PasteButton.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text } from 'react-native'
+import Touchable from 'src/components/Touchable'
+import { Colors } from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { useClipboard } from 'src/utils/useClipboard'
+
+interface Props {
+  onPress?: (clipboardContent: string) => void
```

Comment:
why is this optional? (what is the purpose of a paste button that takes no callback?)

Reply:
It was from before when I had setValue (which was later removed) and onPress.

I've changed it to a required property